docs/kubernetes.md

```diff
@@ -61,7 +61,7 @@ Note that the Kubernetes cluster does not yet support the `--config` flag to cus
 
 To generate a markdown summary of the loadtest results:
 ```
-ltparse results --file $HOME/.mattermost-load-test-ops/myloadtestcluster/results/*.txt --display markdown
+ltparse results --file $HOME/.mattermost-load-test-ops/cluster-name/results/*.txt --display markdown
```

Comment:
Using `cluster-name` here makes it consistent with the rest of the document, and provides for easier copy and paste examples.